src/main/kotlin/org/rucca/cheese/model/TaskSubmissionReviewDetailDTO.kt

```
@@ -0,0 +1,23 @@
+ */
+data class TaskSubmissionReviewDetailDTO(
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("accepted", required = true)
+    val accepted: kotlin.Boolean,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("score", required = true)
+    val score: kotlin.Int,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("comment", required = true)
+    val comment: kotlin.String
+) {}
```

detekt

[warning] The class or object TaskSubmissionReviewDetailDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)